cpp/ql/test/library-tests/ir/memset/MemsetMayBeDeleted.ql

```diff
@@ -0,0 +1,54 @@
+/**
+ * @name Call to `memset` may be deleted
```

Comment:
Why is this query in the test directory, rather than a `.qlref`?  Where is the `.expected` for it?

Reply:
Another question: is "my" version of the `memset` query (https://github.com/Semmle/ql/pull/1933) still necessary?

Reply:
We'll at least want the qhelp from #1933, as well as any tests that didn't make it across.